CWE/common/countries/European Union.txt

```diff
@@ -81,7 +69,7 @@ party = {
 	end_date = 2092.1.1
 }
 party = {
-	name = "party_populist_0_XEU" #Europe of Freedom and Direct Democracy 
+	name = "Identity and Democracy (P)"
 	citizenship_policy = full_citizenship
```

Comment:
Why would this group support full citizenship and secularism?

Reply:
I have no idea - its like that in base CWE and I didn't touch most of the issues except for making market socialists use planned_economy1. I'll change it.